packages/gatsby-theme-docs/src/layouts/internals/layout-page-release-notes-filters.js

```diff
@@ -173,6 +199,7 @@ const LayoutPageReleaseNotesFilters = (props) => {
     return modalPortalNode
       ? ReactDOM.createPortal(
           <Overlay
+            zIndex="22"
```

Comment:
What's this magic number?

Reply:
To be above the top menu, but I see we have something in the designSystem for that 👍🏼